src/stats.rs

```diff
@@ -47,6 +46,10 @@ pub struct GooseStats {
     pub users: usize,
     /// Goose request statistics.
     pub requests: GooseRequestStats,
+    /// Flag indicating whether or not to display percentile.
+    pub display_percentile: bool,
+    /// Flag indicating whether or not to display status_codes.
+    pub display_status_codes: bool,
```

Comment:
Is it ensured that both are false at the start?

Reply:
Yes, because of https://doc.rust-lang.org/std/primitive.bool.html#impl-Default

I should update the comment to make this explicitly clear.

Reply:
Updated:

```rust
    /// Flag indicating whether or not to display percentile. Because we're deriving Default,
    /// this defaults to false.
    pub display_percentile: bool,
    /// Flag indicating whether or not to display status_codes. Because we're deriving Default,
    /// this defaults to false.
    pub display_status_codes: bool,
```